src/createBot.test.ts

```diff
@@ -191,17 +193,24 @@ describe('send functions', () => {
   });
 });
 
+function log(line: string) {
+  fs.appendFileSync('./log.txt', `${line}\n`);
```

Comment:
We need a way to handle failures, i.e. we don't want the entire suite to crash. Can we use a try-catch with this synchronous method, or use await with the non-synchronous method; then handle errors?